client/kaia_client.go

```diff
@@ -472,7 +477,7 @@ func (ec *Client) CallContract(ctx context.Context, msg kaia.CallMsg, blockNumbe
 	if err == nil {
 		return hex, nil
 	}
-	return nil, err
+	return nil, errors.Join(err)
```

Comment:
Is it intentional to ignore the error from `kaia_call` only here?

Reply:
That was wrong. Updated. Thank you